### DISCUSSION TITLE
Update smart playlists produces error "Invalid API request"

### DISCUSSION THREAD

**Discussion by avalx** [jcorporation/myMPD#627] [Q&A] [answered]
Every time I try to do: Main menu -> Update -> Update smart playlists I get an error popup as mentioned above.  The same happens if I go into Settings ->Maintenance -> Force Update. I've checked the mpd log but there is nothing about this. I have Genre based smartlists enabled and they are populated with the songs and I can add them to the queue. So I'm  wondering  what this error might mean, any ideas?

BTW, just discovered myMPD and so far I like it very much!

**Comment by jcorporation** [comment, marked as answer]
Thisis a bug in the 8.1.x release series and already fixed in v8.2.0 (devel).

**Reply by vernonjvs** [reply]
Confirmed fixed in development build. Thanks very much.
